Submitted By:            vvaucoul <vvaucoul@student.42.fr>
Date:                    2022-05-30
Initial Package Version: 1.0.0
Upstream Status:         Not submitted
Origin:                  little-penguin-1
Description:             EXTRAVERSION modification to -thor_kernel

--- bck-Makefile	2022-05-30 14:51:54.622383541 +0200
+++ Makefile	2022-05-30 14:52:42.398381484 +0200
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 18
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION = -thor_kernel
 NAME = Superb Owl
 
 # *DOCUMENTATION*
